spec/routes/api/cli/golden-files/200-vm eu-central-h1_test-vm show -f firewalls.txt

```diff
@@ -0,0 +1,8 @@
+firewall 1:
+  id: fw4gj2v4h1fe3q28q0hnf7g8n1
+  name: default-eu-central-h1-default
+  description: Default firewall
+  location: eu-central-h1
+  path: /location/eu-central-h1/firewall/default-eu-central-h1-default
+  rule 1: fra7jvrz94be7qy8qrq8n1jrp1  0.0.0.0/0  0..65535  
+  rule 2: fr7cv7h5bz324maazy93ykn8s4  ::/0  0..65535  
```

Comment:
This seemed a bit inconsistent to me, compared to other representations. I guess I expected these to be nested, maybe more like this from above:
```
Current metric destinations:
  1: et2t1bswsqz21m7j7njfjbp901  foo  https://baz.example.com
  2: etqhb3wgwvge5pepd72pcgzyp6  foo  https://md.example.com
```
That would result in two layers of indentation though, which I don't think I've seen elsewhere in the goldenfiles (yet anyway), though I may have forgotten or overlooked it.

Reply:
There are other places with multiple levels of nesting.  I don't have a problem doing that, but it saves only 3 characters per line (`s/rule /  /`), so in most cases, doing so would make the output larger even with a significant number of rules.

Reply:
I made this change in https://github.com/ubicloud/ubicloud/pull/2842/commits/d53974e8b2dc057d229d4228be9c7a5a3fd72da7